### PR TITLE
Use ckeditor in config

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
This fields now have an editor:

- legal notice
- welcome text
- ~~users pdf welcome text~~

### CODE REVIEW

File: openslides/users/config_variables.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def get_config_variables():
     yield ConfigVariable(
         name='users_pdf_welcometitle',
         default_value='Welcome to OpenSlides',
+        input_type='markupText',
```
Comment (emanuelschuetze) [inline openslides/users/config_variables.py]: Please remove this line. We have to add html parsing support first for access data pdf. And markup text it's not really required for this pdf.

Comment (FinnStutzenstein) [inline openslides/users/config_variables.py]: Done.

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze ok.

File: openslides/core/static/templates/config-form-field.html [hunk 2]
```diff
@@ -71,6 +71,15 @@
               id="{{ key }}" class="form-control">
     </textarea>
 
+    <!-- editor -->
+    <textarea ng-if="type == 'editor'"
+      id="{{ configOption.key }}"
+      ckeditor="ckeditorOptions"
+      ng-model="$parent.value" class="form-control"
+      ng-model-options="{debounce: 1000}">
+  </textarea>
```
Comment (normanjaeckel) [inline openslides/core/static/templates/config-form-field.html]: Indention broken.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Done and rebased.

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein Please resolve the conflict in CHANGELOG. Thanks.

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Ok. Changed the wording from the second last entry in section core.

**Comment by emanuelschuetze** [comment]
@normanjaeckel Please review and merge.

**Pull request merged**